proxy/src/proxy.rs

```diff
@@ -304,7 +304,7 @@ fn handle_server_event(
             error!(role = "c2p", %error, ?discarded_bytes, "Discard server message");
             return ControlFlow::Continue;
         }
-        Err(ClientFlowError::Io(error)) => {
+        Err(ClientFlowError::Stream(error)) => {
```

Comment:
Should we log `ClientFlowError::Stream(StreamError::Closed)` with a lower log level?

Reply:
Not sure. IMAP is rather specific about how a connection is finalized dictating a logout sequence, the consumption of all unsolicited responses before a TCP close, etc. So, I'm not sure what constitutes an error, really. Shall we open an issue and triage at some later time?

Reply:
I've created issue #86.